man/openrc-run.8

```diff
@@ -119,6 +119,9 @@ The amount of time, in milliseconds, s6-svc should wait for the service
 to go down when stopping the service. The default is 60000.
 .It Ar start_stop_daemon_args
 List of arguments passed to start-stop-daemon when starting the daemon.
+.It Ar supervise_daemon_args
+List of arguments passed to start-stop-daemon when starting the daemon under
```

Comment:
Reword this.  Looks like it was copy/pasted from start_stop_daemon_args and not corrected.

Reply:
@dwfreed done. (though you may have to click on 'Files changed' to see it.